docs/installation.rst

```diff
@@ -139,16 +167,18 @@ how to deploy on various other servers.
 Troubleshooting
 +++++++++++++++
 
-If you are encountering difficulties getting the above to work, it is helpful
-to turn on debugging output. Do this by adding the following line to your
-config file:
+Server errors will be output to the web server's error log by default (in Apache on
+Debian/Ubuntu, for example, this is ``/var/log/apache2/error.log``). Each client
+request will be logged to the web server's access log (in Apache on Debian/Ubuntu
+this is ``/var/log/apache2/access.log``). If you want more debug output then you
+can try activating flask debug mode by adding the following line to your config
```

Comment:
I'm not sure we should be recommending admins turn on Flask's DEBUG mode any more --- does it give anything extra?